build.gradle

```diff
@@ -37,6 +37,7 @@ repositories {
 dependencies {
     testCompile "junit:junit:4.12"
     testCompile "io.dropwizard:dropwizard-testing:${dropwizardVersion}"
+    testCompile "org.mockito:mockito-core:2.+"
```

Comment:
All the other libraries are pinned to a specific version. Not sure if we want to start deviating from that right now.

Reply:
Good point. I was just going off this: https://github.com/mockito/mockito/wiki/Declaring-mockito-dependency

I added a commit to make the version more specific.